src/tool/print.tool.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  BlockchainNetworkEnum,
+  IAssetBalance,
+} from '@kitzen/data-transfer-objects';
+import type { IAssetMetadata } from '@kitzen/assets';
+
+export type CoinIdentifier = Pick<IAssetBalance, 'identifier' | 'network'>;
+export type AssetIdentifier = Pick<IAssetBalance, 'identifier' | 'balance' | 'network'>;
+export type AssetIdentifierWithRate = Pick<IAssetBalance, 'identifier' | 'balance' | 'network' | 'rate'>;
+
+export class PrintTool {
```

Comment:
This tools is different from Calculation tool. I need a class that I can tell, here's the asset, just give me something I can print to the screen